docs/docusaurus/docs/cloud/set_up_gx_cloud.md

```diff
@@ -72,18 +72,3 @@ Environment variables securely store your GX Cloud access credentials. The GX Cl
 
     To edit an environment variable, stop the GX Cloud agent, edit the environment variable, save the change, and then restart the GX Cloud agent.
 
-## Troubleshoot
-
-Use the following information to help resolve issues with the GX Cloud agent installation.
-
-**Error: `great_expectations.agent.agent.GXAgentError: Missing or badly formed environment variable`**
-
-Confirm the `GX_CLOUD_ACCESS_TOKEN` and `GX_CLOUD_ORGANIZATION_ID` environment variables are set and available in your local environment. For example, on Unix systems, you can run `echo $GX_CLOUD_ORGANIZATION_ID` to confirm that your organization ID is present and set.
-
-**Error: `pika not found`**
-
-Run the following command:
-
-```bash title="Terminal input"
-    pip install 'great_expectations[cloud,snowflake]'
-```
```

Comment:
@kwcanuck You said you removed this content because it's no longer relevant. Can you explain that?

Reply:
@Kilo59, it was my understanding (perhaps incorrect) that the environment variable error message wasn't relevant to the Docker methodology.  If this is not correct, it's easy to add the content back.